### PR TITLE
[ISSUE #4718] Update DISCLAIMER

### PR DESCRIPTION
**Pull request by ZhengLin-Li**
Fixes https://github.com/apache/eventmesh/issues/4718

### Motivation

EventMesh is no longer in incubator

### CODE REVIEW

File: DISCLAIMER [hunk 1]
```diff
@@ -1,11 +1,5 @@
-Apache EventMesh is an effort undergoing incubation at The Apache Software Foundation (ASF),
-sponsored by the Apache Incubator PMC.
-
-Incubation is required of all newly accepted projects until a further review indicates that the infrastructure, 
-communications, and decision-making process have stabilized in a manner consistent with other successful ASF projects.
-
-While incubation status is not necessarily a reflection of the completeness or stability of the code, 
-it does indicate that the project has yet to be fully endorsed by the ASF.
+Apache EventMesh, previously incubating at The Apache Software Foundation (ASF) 
+has successfully graduated from the incubator phase.
```
Comment (Pil0tXia) [inline DISCLAIMER:3]: You may take other apache project's DISCLAIMER file as a reference.

Comment (ZhengLin-Li) [reply DISCLAIMER:3]: I noticed that not many apache projects have DISCLAIMER file,

Comment (Pil0tXia) [reply DISCLAIMER:3]: A podling project must include a disclaimer in its repo, and a TLP doesn't have a disclaimer policy according to https://incubator.apache.org/policy/incubation.html#disclaimers. So you can just delete this file.

Comment (Pil0tXia) [reply DISCLAIMER:3]: The word DISCLAIMER also appears twice in the `.licenserc.yaml` file and can be removed.

Comment (ZhengLin-Li) [reply DISCLAIMER:3]: > A podling project must include a disclaimer in its repo, and a TLP doesn't have a disclaimer policy according to https://incubator.apache.org/policy/incubation.html#disclaimers. So you can just delete this file.

sure, thanks for you guidance. then i will delete this file

**Comment by ZhengLin-Li** [comment]
hi @Pil0tXia  thanks for your review, i have updated the PR.

**Review by Pil0tXia — APPROVED** [review]
LGTM

**Review by mxsm — APPROVED** [review]
LGTM

**Pull request merged**